errors.toml

```diff
@@ -98,7 +98,7 @@ rewrite rule not found
 
 ["BR:KV:ErrKVUnknown"]
 error = '''
-unknown tikv error
+unknown error occur on tikv, please ensure tikv is permitted to read from & write to backup storage.
```

Comment:
Seems the error can occur in restore too, maybe 'the storage' would be enough.

(BTW, I'm afraid other errors might be treated as `BR:KV:ErrKVUnknown` too, at such condition, maybe this message becomes misleading. Thankfully, seems currently there isn't.)